tests/sentry/api/endpoints/test_relay_projectconfigs.py

```diff
@@ -169,7 +169,12 @@ def test_relays_dyamic_sampling(client, call_endpoint, default_project, dyn_samp
     """
     default_project.update_option("sentry:dynamic_sampling", dyn_sampling_data())
 
-    with Feature({"organizations:server-side-sampling": True}):
+    with Feature(
+        {
+            "organizations:server-side-sampling": True,
```

Comment:
I think we can now even remove `server-side-sampling` here now, right?

Reply:
correct but we decided with @matejminar and @ahmedetefy that we can do it in follow up PR.

Reply:
Ah, the multiplexer still checks server-side-sampling, sorry, did not realize that.